packages/amplication-server/src/core/pluginCatalog/pluginCatalog.service.ts

```diff
@@ -0,0 +1,103 @@
+import { Injectable } from "@nestjs/common";
+import { ApolloClient, InMemoryCache, gql } from "@apollo/client";
```

Comment:
To avoid any React dependency, import the apollo client from the core instead.
```suggestion
import { ApolloClient, InMemoryCache, gql } from "@apollo/client/core";
```

Reply:
nice!